src/Atmos/Model/AtmosModel.jl

```diff
@@ -63,7 +63,7 @@ function vars_diffusive(m::AtmosModel, T)
 end
 function vars_aux(m::AtmosModel, T)
   @vars begin
-    coord::@vars(x::T,y::T,z::T)
+    coord::::SVector{3,T}
```

Comment:
```suggestion
    coord::SVector{3,T}
```

Reply:
Thanks, i should make hasty PRs while running out the door.